hack/test_e2e.sh

```diff
@@ -23,6 +23,9 @@ shopt -s expand_aliases
 OS=$(uname| tr '[:upper:]' '[:lower:]')
 CONTAINER_ENGINE="docker"
 
+# Users can specify docker.io, quay.io registry
+KINDEST_NODE_IMAGE="docker.io/kindest/node"
```

Comment:
Nice! Should we also allow this to be a command line arg with default set to docker.io? What do you think?

Reply:
I agree but in the first place I would ship as static. In the next weeks we can add such feature as well.